res/css/structures/_RoomStatusBar.pcss

```diff
@@ -186,7 +186,7 @@ limitations under the License.
 
 .mx_RoomStatusBar_typingBar {
     height: 50px;
-    line-height: $font-50px;
+    line-height: 50px;
```

Comment:
Won't this diff change how appearance font scaling affects the typing bar? A fixed height is unlikely to work well, no?

Reply:
It's done this way because the `height` above is also defined in pixels. The `line-height` is set as such so things are centered vertically.
Zooming does not create major issues. Font scaling does not work great as the `height` is set in an absolute value, but the issue does exist today too